.travis.yml

```diff
@@ -60,7 +60,7 @@ after_success:
   - |
     if [ "$TRAVIS_JOB_NAME" = "Ubuntu 14.04 JDK 8" ];
     then
-      git fetch --all && git config --global user.email "-" && git config --global user.name "-" && ./gradlew run -Dargs="--since d1"
+      git fetch --all && git config --global user.email "-" && git config --global user.name "-" && ./gradlew run -Dargs="--since d1 --config ./config/"
```

Comment:
after changing the default location, this line edition is no longer needed right?

Reply:
@Tejas2805

Reply:
Doesn't make a difference, true. Will remove.